CMakeLists.txt

```diff
@@ -381,6 +381,7 @@ endif(APPLE)
 
 # OpenBSD
 if(OPENBSD)
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -L/usr/local/lib")
```

Comment:
If you have linking issues this may indicate that the libraries detected by CMake have not properly been added. Overwriting ```C_FLAGS``` is not the way to go.